src/content/posts/2023-01-31-pledge-to-the-ethereum-classic-community-with-regard-to-the-eth_classic-twitter-account-en/2023-01-31-index.en.md

```diff
@@ -8,9 +8,9 @@ tags: [communications, pledge]
 
 Various community members have stated that they would like to see some official pledge from the ETC Cooperative with regard to the new [eth_classic](https://twitter.com/eth_classic) Twitter account which is currently in possession of its executive director, [Bob Summerwill](https://etccooperative.org/people). 
 
-The background of this situation is that the previous account with the same handle had more than 600,000 followers, but it was in control of [Charles Hoskinson](https://iohk.io/en/team/charles-hoskinson), the founder of a firm called IOHK. After a fallout between Hoskinson and the ETC ecosystem, the account remained inactive for a long time and was eventually transferred to the community of another project of the company. This stripped Ethereum Classic from one of its most valuable social media assets, so it is only reasonable that the new @eth_classic account be managed under a sort of social contract in the form of this pledge to provide some guarantees that events like the mentioned here will not happen again.
```

Comment:
Here I would say ...

the account remained inactive for a long time and was eventually transferred by Charles to the community of another project (Ergo) which he actively promotes

Reply:
Perfect. Changing now.